app/src/components/Modals/add-activity-modal.tsx

```diff
@@ -254,21 +254,21 @@ const AddActivityModal: FC<AddUserModalProps> = ({
               >
                 <FormControl className="w-full">
                   <BuildingTypeSelectInput
-                    options={formInputs?.fields[0].options || []}
-                    title={formInputs?.fields[0].label}
+                    options={[]}
+                    title={""}
                     placeholder={t("select-type-of-building")}
                     register={register}
-                    activity={"activity." + formInputs?.fields[0].name}
+                    activity={"activity."}
                     errors={errors}
                   />
                 </FormControl>
                 <FormControl>
                   <BuildingTypeSelectInput
-                    options={formInputs?.fields[1].options}
-                    title={formInputs?.fields[1].label}
+                    options={[]}
+                    title={""}
```

Comment:
This should come from the json file and be translated with a `t()` call (individually)